src/test/java/plugins/MavenPluginTest.java

```diff
@@ -246,4 +248,20 @@ private void assertHasModule(MavenModuleSet job, String name) {
         find(by.xpath("//a[@href='%s/']", name)).click();
         assertThat(driver.getCurrentUrl(), equalTo(build.module(name).url.toExternalForm()));
     }
+
+    @Issue("JENKINS-22252")
+    @WithPlugins({"maven-plugin@2.12", "tasks"})
```

Comment:
It should be the first version where necessary features was introduced. As it is a regression introduced in 2.12 IIUC, there should be no version specified (presuming it does not depend on any recent features).

As a rule of thumb, consider someone is running this against a plugin version with `JENKINS-22252` present. If we declare this should run with first *fixed* version we effectively **hide** the presence of the bug from the user instead of pointing his attention to it.